safe/utilities/memory_checker.py

```diff
@@ -166,3 +166,24 @@ def check_memory_usage(buffered_geo_extent, cell_size):
     send_dynamic_message(dispatcher.Anonymous, message)
     # LOGGER.info(message.to_text())
     return True
+
+
+def memory_error():
+    """Display an error when there is not enough memory."""
+    warning_heading = m.Heading(
+        tr('Memory issue'), **WARNING_STYLE)
+    warning_message = tr(
+        'There is not enough free memory to run this analysis.')
+    suggestion_heading = m.Heading(
+        tr('Suggestion'), **INFO_STYLE)
```

Comment:
I believe there is SUGGESTION_STYLE in the styles.py. Perhaps it's more suitable to use that.